sigar_darwin.go

```diff
@@ -422,6 +422,10 @@ func task_info(pid int, info *C.struct_proc_taskallinfo) error {
 	return nil
 }
 
+func (self *SystemDistribution) Get() error {
+	return notImplemented()
```

Comment:
Right now we wrap all the SIGAR calls with `recover`. But this is some tech debt, it'd be great to remove all the panics and produce proper error messages.